packages/react-resizable-panels/src/PanelResizeHandleRegistry.ts

```diff
@@ -73,9 +73,48 @@ export function registerResizeHandle(
     if (count === 1) {
       ownerDocumentCounts.delete(ownerDocument);
     }
+
+    recalculateIntersectingHandlesAfterUnregister(element);
   };
 }
 
+function recalculateIntersectingHandlesAfterUnregister(
```

Comment:
@bvaughn 

These changes were written by ChatGPT after I explained the problem to it and fed it the source code for the relevant files. It seems to work as expected; I've not yet observed the issue re-appear in the project I have linked against the forked version of this library. It seems to match your recommended approach:

> I think the right fix might be- on unregister- if there are any current intersecting drag handles, re-compute them and then update the cursor again (using the same methods already defined in that module)

I asked ChatGPT (and had a look myself) whether it would be possible to call the existing `recalculateIntersectingHandles` function within the unregister callback, but it seems this would require more refactoring and potentially be more brittle than the solution ChatGPT proposes that involves writing these two new functions. We can discuss whether there's a way to simplify this further and re-use more of the existing code, though.

Reply:
> These changes were written by ChatGPT

I remember why I didn't re-review this. This was kind of a turn off to me. In my experience, ChatGPT generates code that looks plausible but is often wrong in som way.

In this case, I don't think this new function is necessary. It's called by `unregisterResizeHandle` **after** the unmounting handle has been removed from `registeredResizeHandlers` (so that special case check is not needed). Given that, wouldn't the `recalculateIntersectingHandles` method work for updating the intersecting handles?

We'd still want to call `updateCursor` afterward but it will already call `resetGlobalCursorStyle` if there are no intersecting handles.